codegen-server/src/test/kotlin/software/amazon/smithy/rust/codegen/server/smithy/protocols/EventStreamTestTools.kt

```diff
@@ -98,7 +98,7 @@ private fun fillInBaseModel(
         MessageWithNoHeaderPayloadTraits: MessageWithNoHeaderPayloadTraits,
         SomeError: SomeError,
     }
-    structure TestStreamInputOutput { @required value: TestStream }
+    structure TestStreamInputOutput { @httpPayload @required value: TestStream }
```

Comment:
What was this change for?

Reply:
[This](https://github.com/awslabs/smithy/pull/1076) and possibly another breakfix for model handling. We needed to make the same fix for Kotlin code generation as well. Only http protocol test instantiations of this model require the trait, but the non-http protocol test instantiations ignore it so it's safe to add like this.